utils/data.py

```
@@ -0,0 +1,31 @@
+
+DATA_DIR = "data"
+
+
+def load_data(filename):
+    file_path = os.path.join(DATA_DIR, filename)
+
+    if not os.path.exists(file_path):
+        return []
+
+    try:
+        with open(file_path, "r", encoding="utf-8") as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)